File: lecture_notes/08 collections/8-map.js
ages = {}

ages["Vasia Pupkin"] = 19
ages["Marcus Aurelius"] = 1860
print(ages)

ages["Vasia Pupkin"] = 20
print(ages)

ages.pop("Vasia Pupkin")
print(ages)

print(f"Vasia Pupkin: {'Vasia Pupkin' in ages}, Marcus Aurelius: {'Marcus Aurelius' in ages}")

ages.clear()
print("Prints an open dictionary: ", ages)

File: lecture_notes/08 collections/8-map.py
ages = {
    'Vasia Pupkin': 19,
    'Marcus Aurelius': 1860,
}
print(f'ages: {ages}')

ages['Vasia Pupkin'] = 20
print(f'ages: {ages}')

ages.pop('Vasia Pupkin')
print(f'ages: {ages}')

print({
    'Vasia Pupkin': 'Vasia Pupkin' in ages,
    'Marcus Aurelius': 'Marcus Aurelius' in ages,
})

ages.clear()
print(ages)
